Content.Server/StationEvents/Components/SpawnRuleComponent.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Content.Server.StationEvents.Events;
+
+namespace Content.Server.StationEvents.Components;
+
+/// <summary>
+/// Tarantula spawning component
+/// <summary>
+[RegisterComponent, Access(typeof(SpiderSpawnRule))]
+public sealed class SpiderSpawnRuleComponent : Component
+{
+
+}
+
+/// <summary>
+/// Slime spawning component
+/// <summary>
+[RegisterComponent, Access(typeof(SlimesSpawnRule))]
+public sealed class SlimesSpawnRuleComponent : Component
+{
+    [DataField("spawnedPrototypeChoices")]
+    public List<string> SpawnedPrototypeChoices = new()
+    {
+
+    };
+}
```

Comment:
I'll just fix this when I get a chance but the excess sprites will still need removing as we don't take placeholders (without good reason).

Reply:
Okay, I'll remove the extra sprites for now. I'll add them while I'm using them.